shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -19,6 +19,20 @@
 #import "flutter/shell/platform/darwin/ios/ios_surface.h"
 #import "flutter/shell/platform/darwin/ios/ios_surface_gl.h"
 
+@implementation UIView (FirstResponder)
+- (BOOL)hasFirstResponderInViewHierarchySubtree {
+  if (self.isFirstResponder) {
+    return YES;
+  }
+  for (UIView* subview in self.subviews) {
+    if (subview.hasFirstResponderInViewHierarchySubtree) {
```

Comment:
nits:
Use method call syntax instead of . notation for methods without arguments to avoid confusion. 
(For example, dot notation would lead reader to try to find a property named `hasFirstResponderInViewHierarchySubtree` in UIView.